web/gridt/tests/integration/resources/test_movements.py

```diff
@@ -200,23 +200,18 @@ def test_name_empty(self):
             )
             self.assertEqual(resp.status_code, 400)
 
-    def test_post_sucessful(self):
+    def test_post_successful(self):
         with self.app_context():
-            user = User("test1", "test@test.com", "pass")
-            user.save_to_db()
-
-            token = self.obtain_token("test@test.com", "pass")
+            user = self.create_user()
 
             movement_dict = {
                 "name": "movement",
                 "short_description": "Hi, hello this is a test",
                 "interval": "daily",
             }
 
-            resp = self.client.post(
```

Comment:
The difference here is simply that we now associate a user with the request instead of just doing an anonimous request right?

Reply:
Actually it's the same code, but instead of doing it manually, a helper function is called to perform the request.